src/main/java/com/minecolonies/coremod/network/messages/server/BuildToolPlaceMessage.java

```diff
@@ -200,6 +203,14 @@ private static void handleHut(
       final boolean mirror,
       final BlockState state)
     {
+        final Block blockAtPos = world.getBlockState(buildPos).getBlock();
+        if (blockAtPos instanceof IBuilderUndestroyable || ModTags.indestructible.contains(blockAtPos))
+        {
+            LanguageHandler.sendPlayerMessage(player, IMMUTABLE_BLOCK_AT_POS);
+            SoundUtils.playErrorSound(player, buildPos);
+            return;
```

Comment:
wouldnt this also prevent pasting over an existing building?

Reply:
No, only placing, pasting is a different message